tests/virtctl/create_vm.go

```diff
@@ -237,7 +243,7 @@ func setFlag(flag, parameter string) string {
 }
 
 func runCmd(args ...string) ([]byte, error) {
-	_args := append([]string{CREATE, VM}, args...)
+	_args := append([]string{create}, args...)
```

Comment:
Can't you keep `VM` here? Then you don't need to add it to the runCmd calls.

Reply:
I have removed this function as we have discussed.